middleware/parse/parse.go

```diff
@@ -73,6 +74,9 @@ func (m *Middleware) Handle(next goyave.Handler) goyave.Handler {
 					var body any
 					if err := json.Unmarshal(bodyBytes, &body); err != nil {
 						response.Status(http.StatusBadRequest)
+						r.Extra[goyave.ExtraRequestError{}] = []error{
+							fmt.Errorf("request.json-invalid-body"),
```

Comment:
You should use a sentinel, but also, you shouldn't use fmt.Errorf if you are not formatting.

So please use errors.New and move it at the package level, so you can catch it with errors.Is

Reply:
Don't use Goyave error wrapping for this those. Collecting the caller frames isn't useful here. You can use the std errors package.

Reply:
@System-Glitch I already refactored some. Did you check?